spaces_test.go

```diff
@@ -0,0 +1,30 @@
+package cinnabot
+
+import (
+	"testing"
+
+	"gopkg.in/telegram-bot-api.v4"
+)
+
+func Test290516(t *testing.T) {
```

Comment:
Rename test function to describe what the test does. Also, unit tests should be very quick. If it takes more than 100ms, its a bad idea. (Tip - we don't have to test the actual server's response.)